kernel/src/grant.rs

```diff
@@ -382,14 +710,15 @@ impl<'a, T: Default> ProcessGrant<'a, T> {
     /// `Some(fun())`.
     pub fn try_enter<F, R>(self, fun: F) -> Option<R>
     where
-        F: FnOnce(&mut GrantMemory<T>) -> R,
+        F: FnOnce(&mut GrantMemory<T>, &mut UpcallMemory) -> R,
```

Comment:
From call:

GrantMemory -> GrantData
UpcallMemory -> GrantUpcallTable

Reply:
Changed UpcallMemory, other change is in 2651.